isisd/isisd.c

```diff
@@ -2512,7 +2512,7 @@ static void common_isis_summary_vty(struct vty *vty, struct isis *isis)
 		pdu_counter_print(vty, "    ", area->pdu_rx_counters);
 
 		vty_out(vty, "  Advertise high metrics: %s\n",
-				area->advertise_high_metrics ? "Enabled" : "Disabled");
+			area->advertise_high_metrics ? "Enabled" : "Disabled");
 
```

Comment:
I believe this is in the correct commit since I modify CLI and show output here. Did you want me to create a separate commit for changes to show output?

Reply:
Updated